vmdb/app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -36,7 +37,14 @@ def save_container_services_inventory(ems, hashes, target = nil)
                 []
               end
 
-    save_inventory_multi(:container_services, ems, hashes, deletes, [:ems_ref], [:labels, :selector_parts])
+    hashes.each do |h|
+      h[:container_group_ids] = h[:container_groups].map do |x|
+        x.fetch_path(:id)
```

Comment:
`fetch_path` is not needed here.  Prefer `[]` access.  When you do that, I think this line becomes short enough to not need multi-lining.